cluster-cm/src/test/java/com/sequenceiq/cloudbreak/cm/polling/task/ClouderaManagerHostHealthyStatusCheckerTest.java

```diff
@@ -0,0 +1,195 @@
+package com.sequenceiq.cloudbreak.cm.polling.task;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import org.assertj.core.util.Sets;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.cloudera.api.swagger.CommandsResourceApi;
+import com.cloudera.api.swagger.HostsResourceApi;
+import com.cloudera.api.swagger.client.ApiClient;
+import com.cloudera.api.swagger.client.ApiException;
+import com.cloudera.api.swagger.model.ApiCommissionState;
+import com.cloudera.api.swagger.model.ApiHealthSummary;
+import com.cloudera.api.swagger.model.ApiHost;
+import com.cloudera.api.swagger.model.ApiHostList;
+import com.sequenceiq.cloudbreak.cluster.service.ClusterEventService;
+import com.sequenceiq.cloudbreak.cm.client.ClouderaManagerApiPojoFactory;
+import com.sequenceiq.cloudbreak.cm.polling.ClouderaManagerCommandPollerObject;
+import com.sequenceiq.cloudbreak.domain.stack.Stack;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClouderaManagerHostHealthyStatusCheckerTest {
+
+    private static final String VIEW_TYPE = ClouderaManagerHostHealthyStatusChecker.VIEW_TYPE;
+
+    @Mock
+    private ClouderaManagerApiPojoFactory clouderaManagerApiPojoFactory;
+
+    @Mock
+    private ClusterEventService clusterEventService;
+
+    @Mock
+    private HostsResourceApi hostsResourceApi;
+
+    private ClouderaManagerHostHealthyStatusChecker underTest;
+
+    @Before
+    public void init() {
+        when(clouderaManagerApiPojoFactory.getHostsResourceApi(any(ApiClient.class))).thenReturn(hostsResourceApi);
+    }
+
+    @Test
+    public void testSingleGoodHostOnFirstInvocation() throws ApiException {
+        Set<String> hostnamesToCheckFor = Sets.newHashSet(Arrays.asList("h1"));
+        underTest = new ClouderaManagerHostHealthyStatusChecker(clouderaManagerApiPojoFactory, clusterEventService, hostnamesToCheckFor);
+
+        ApiHostList apiHostList = new ApiHostList().items(List.of(
+                constructApiHost("h1", underTest.start.plusMillis(1), ApiHealthSummary.GOOD, false, ApiCommissionState.COMMISSIONED)));
+
+        when(hostsResourceApi.readHosts(null, null, VIEW_TYPE)).thenReturn(apiHostList);
+        boolean result = underTest.doStatusCheck(getPollerObject(), mock(CommandsResourceApi.class));
+        assertEquals(true, result);
```

Comment:
assertTrue

Reply:
Fixed here and elsewhere in the same class (in the next PR that is).